UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -22,7 +22,7 @@
   SUPPORTED_ARCHITECTURES             = IA32|X64
   BUILD_TARGETS                       = DEBUG|RELEASE|NOOPT
   SKUID_IDENTIFIER                    = DEFAULT
-  OUTPUT_DIRECTORY                    = Build/UefiPayloadPkgX64
+  OUTPUT_DIRECTORY                    = Build/UefiPayloadPkg

```

Comment:
Due to some reason for backward, Could you use Build/UefiPayloadPkg$(ARCH) and keep default ARCH = X64

Reply:
I modified in such a way that it will create correct dir based on arch. No need to define a default etc.

Reply:
Some customer has depends on this build folder path, use Build/UefiPayloadPkg$(ARCH) is follow up backward support.

Reply:
Did you check the latest patch I submitted? It does not work?

Reply:
It failure on internal code.

We depends on 

run UefiPayloadPkg/UniversalPayloadBuild.py
  output Build\UefiPayloadPkgX64\UniversalPayload.fit

We internal use code path to patch some Fv into Build\UefiPayloadPkgX64\UniversalPayload.fit.

Reply:
It's after your change.
![image](https://github.com/tianocore/edk2/assets/96405988/662a7ac8-4f9b-4b9e-83ac-964f6fb602c9)

But I hope like below
py -3 UefiPayloadPkg\UniversalPayloadBuild.py -t VS2019 -f -a IA32
Build\UefiPayloadPkgIA32\UniversalPayload.fit

py -3 UefiPayloadPkg\UniversalPayloadBuild.py -t VS2019 -f -a X64
Build\UefiPayloadPkgX64\UniversalPayload.fit

py -3 UefiPayloadPkg\UniversalPayloadBuild.py -t VS2019 -f -a RISV64
Build\UefiPayloadPkgRISV64\UniversalPayload.fit